src/main/java/beam/analysis/summary/VehicleMilesTraveledAnalysis.java

```diff
@@ -46,8 +46,7 @@ public void resetStats() {
                 e -> e.getValue() * 0.000621371192 // unit conversion from meters to miles
         ));
 
-        vehicleTypes.foreach(vt -> result.merge("vehicleMilesTraveled_" + vt.toString(),0D, (d1, d2) -> d1 + d2));
-
+        vehicleTypes.foreach(vt -> result.put("vehicleMilesTraveled_" + vt.toString(),milesTraveledByVehicleType.getOrDefault(vt.toString(), 0.0) * 0.000621371192));
```

Comment:
We should use a variable for the conversion rate. Especially since it's used more than once. Also, this is quite a change n that it doesn't just change the conversion factor - it also overwrites any old values instead of summing them. That seems incorrect, also?

Reply:
Hi @JustinPihony , on line number https://github.com/LBNL-UCB-STI/beam/blob/e8717bd0687bbccab616e5ce9dba3d20b05cd967/src/main/java/beam/analysis/summary/VehicleMilesTraveledAnalysis.java#L30 we already merging length with vehicle type, here we are just preparing stats param,previously we are always merging 0 (look into merge value id 0D) in this stats params so these stats param value was 0

Reply:
Duh - sorry for missing that.